src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildALibraryWithOSSupportedVersion.cs

```diff
@@ -213,7 +213,7 @@ public void WhenTargetingWindowsSupportedOSPlatformVersionPropertyIsPreferredOve
         }
 
         [Theory]
-        [InlineData("netcoreapp3.1")]
+        [InlineData(ToolsetInfo.CurrentTargetFramework)]
```

Comment:
This should not be updated.  The test is testing pre .NET 5 behavior for how target platforms were handled.